service/src/main/java/uk/gov/hmcts/reform/fpl/service/OthersService.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.Others;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static net.logstash.logback.encoder.org.apache.commons.lang3.ObjectUtils.defaultIfNull;
+import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
+
+@Service
+public class OthersService {
+    private static String EMPTY_PLACEHOLDER = "BLANK - Please complete";
+
+    public String buildOthersLabel(Others others) {
+        StringBuilder sb = new StringBuilder();
+
+        if (otherExists(others)) {
+            if (isNotEmpty(others.getFirstOther())) {
+                sb.append("Person 1")
+                    .append(" ")
+                    .append("-")
+                    .append(" ")
+                    .append(defaultIfNull(others.getFirstOther().getName(), EMPTY_PLACEHOLDER))
+                    .append("\n");
+            }
+
+            if (isNotEmpty(others.getAdditionalOthers())) {
+                AtomicInteger i = new AtomicInteger(1);
+
+                others.getAdditionalOthers().forEach(other -> {
+                    sb.append("Other person")
+                        .append(" ")
+                        .append(i)
+                        .append(" ")
+                        .append("-")
+                        .append(" ")
+                        .append(defaultIfNull(other.getValue().getName(), EMPTY_PLACEHOLDER))
+                        .append("\n");
+
+                    i.incrementAndGet();
+                });
+            }
+
+        } else {
+            sb.append("No others on the case");
+        }
+
+        return sb.toString();
+    }
+
+    private boolean otherExists(Others others) {
+        return isNotEmpty(others) && isNotEmpty(others.getFirstOther())
```

Comment:
I personally do not think `isNotEmpty` is right language. It unnecessarily adds to the confusion as empty word fits more string instances or collections. Using that kind of language confuses me personally especially in places where type information is not available at hand. Simple `!= null` is way better for me.

Reply:
I have swapped all instances of isNotEmpty -> != null in othersService